packages/components/stories/button-link/README.md

```diff
@@ -0,0 +1,62 @@
+# ButtonLink Component
+
+Renders `ButtonLink` component.
```

Comment:
Isn't it better to avoid starting with `renders`?. We all know components render something, So I think it's better to just describe the component.